### PR TITLE
[AIRFLOW-252] Raise Sqlite exceptions when deleting tasks instance in WebUI

### PR DESCRIPTION
**Pull request by sekikn**
Dear Airflow Maintainers,

Please accept this PR that addresses the following issues:
- https://issues.apache.org/jira/browse/AIRFLOW-252

Testing Done:
- all 43 core unit tests passed
- confirmed delete button in "Task Instances" view worked with both SQLite and MySQL

If users who use SQLite as backend try to delete a task via browser, it fails with an exception.
Though this is a bug on Flask-Admin's side basically, this patch provides a workaround for it.

### CODE REVIEW

File: airflow/www/views.py [hunk 1]
```diff
@@ -2236,6 +2237,17 @@ def set_task_instance_state(self, ids, target_state, session=None):
                 raise Exception("Ooops")
             flash('Failed to set state', 'error')
 
+    def get_one(self, id):
+        """
+            As a workaround for AIRFLOW-252, this method overrides Flask-Admin's ModelView.get_one().
```
Comment (criccomini) [inline airflow/www/views.py]: Nit: no extra tab spacing needed here. Should align with """

**Comment by criccomini** [comment]
LGTM except for the spacing issue. If you correct, I can merge.

**Comment by sekikn** [comment]
Thanks! Updated the PR.

**Comment by jkryanchou** [comment]
👍  awesome.

**Pull request merged**